### PR TITLE
Fixes #15381 - Added help to capsule-remove

### PR DESCRIPTION
**Pull request by chris1984**
Added help to capsule-remove

### CODE REVIEW

File: bin/capsule-remove [hunk 1]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+for var in $@; do
+   if [[ $var == "--help" || $var == "-h" ]]
+       then echo -e "This script is used to remove all capsule related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
+       exit 1
+     fi
```
Comment (ehelms) [inline bin/capsule-remove]: Indentation is off here.

File: bin/capsule-remove [hunk 2]
```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+for var in $@; do
+   if [[ $var == "--help" || $var == "-h" ]]
+       then echo -e "This script is used to remove all capsule related packages and services. \nThis should only be used if you are sure you are wanting to completly remove Katello from this machine and lose all of your settings and synced content."
```
Comment (ehelms) [inline bin/capsule-remove]: s/Katello/Capsule since this is capsule remove :)

**Comment by chris1984** [comment]
@ehelms thanks for looking at it and providing the needed fixes. I have changed Katello to Capsule and fixed the spacing on the if statement.

**Comment by ehelms** [comment]
Thanks @chris1984

**Pull request merged**